models/_sector/alm/alm_base_trades.sql

```diff
@@ -0,0 +1,67 @@
+{{ config(
+    schema = 'dex'
+    , alias = 'base_lps'
+    , partition_by = ['block_month', 'blockchain', 'project']
+    , materialized = 'incremental'
+    , file_format = 'delta'
+    , incremental_strategy = 'merge'
+    , unique_key = ['blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'vault_address']
```

Comment:
I think we can make the roll-up spells materialized as `view`.

Reply:
i think it'd be best to materialize the final downstream spell incrementally, then have the chain-level roll-up spell(s) be views. for this PR, that would be the `alm_ethereum_base_trades` as a view